src/kernels/Conv_Winograd_v13_3_12_fp16dot_stride1.inc

```diff
@@ -23,2975 +23,3038 @@
+s_add_u32 s92, s92, 1
+s_addc_u32 s93, s93, 0
+v_mov_b32_e32 v4, s8
+v_mov_b32_e32 v5, s17
+v_and_b32_e32 v6, 3, v0
+v_cmp_eq_u32_e32 vcc, 2, v6
+v_cndmask_b32_e32 v4, v4, v5, vcc
+v_cmp_eq_u32_e32 vcc, 1, v6
+v_cndmask_b32_e32 v7, 0, v107, vcc
+v_cmp_eq_u32_e64 s[48:49], 3, v6
+v_bfe_u32 v105, v7, 0, 5
+v_mad_u32_u24 v105, v4, 32, v105
+v_cvt_f32_u32_e32 v8, s45
+v_rcp_f32_e32 v8, v8
+v_mul_f32_e32 v8, 0x4f800000, v8
+v_cvt_u32_f32_e32 v8, v8
+v_mul_lo_u32 v9, s45, v8
+v_mul_hi_u32 v10, s45, v8
+v_sub_co_u32_e32 v11, vcc, 0, v9
+v_cmp_ne_i32_e64 s[50:51], 0, v10
+v_cndmask_b32_e64 v9, v11, v9, s[50:51]
+v_mul_hi_u32 v9, v9, v8
+v_sub_co_u32_e32 v10, vcc, v8, v9
+v_add_co_u32_e32 v8, vcc, v8, v9
+v_cndmask_b32_e64 v8, v8, v10, s[50:51]
+v_mul_hi_u32 v8, v8, v105
+v_mul_lo_u32 v9, v8, s45
+v_sub_co_u32_e32 v10, vcc, v105, v9
+v_cmp_ge_u32_e64 s[50:51], v105, v9
+v_cmp_ge_u32_e64 s[52:53], v10, s45
+v_add_co_u32_e32 v10, vcc, 1, v8
+s_and_b64 s[52:53], s[50:51], s[52:53]
+v_add_co_u32_e32 v9, vcc, -1, v8
+v_cndmask_b32_e64 v10, v8, v10, s[52:53]
+v_cndmask_b32_e64 v10, v9, v10, s[50:51]
+v_cmp_ne_i32_e64 vcc, 0, s45
+v_cndmask_b32_e32 v106, -1, v10, vcc
+v_mad_i32_i24 v104, v106, s78, v105
+v_lshrrev_b32_e32 v105, 5, v7
+v_mad_u32_u24 v105, v106, 1, v105
+v_cndmask_b32_e64 v105, v105, 1, s[48:49]
+v_cvt_f32_u32_e32 v8, s44
+v_rcp_f32_e32 v8, v8
+v_mul_f32_e32 v8, 0x4f800000, v8
+v_cvt_u32_f32_e32 v8, v8
+v_mul_lo_u32 v9, s44, v8
+v_mul_hi_u32 v10, s44, v8
+v_sub_co_u32_e32 v11, vcc, 0, v9
+v_cmp_ne_i32_e64 s[50:51], 0, v10
+v_cndmask_b32_e64 v9, v11, v9, s[50:51]
+v_mul_hi_u32 v9, v9, v8
+v_sub_co_u32_e32 v10, vcc, v8, v9
+v_add_co_u32_e32 v8, vcc, v8, v9
+v_cndmask_b32_e64 v8, v8, v10, s[50:51]
+v_mul_hi_u32 v8, v8, v105
+v_mul_lo_u32 v9, v8, s44
+v_sub_co_u32_e32 v10, vcc, v105, v9
+v_cmp_ge_u32_e64 s[50:51], v105, v9
+v_cmp_ge_u32_e64 s[52:53], v10, s44
+v_add_co_u32_e32 v10, vcc, 1, v8
+s_and_b64 s[52:53], s[50:51], s[52:53]
+v_add_co_u32_e32 v9, vcc, -1, v8
+v_cndmask_b32_e64 v10, v8, v10, s[52:53]
+v_cndmask_b32_e64 v10, v9, v10, s[50:51]
+v_cmp_ne_i32_e64 vcc, 0, s44
+v_cndmask_b32_e32 v106, -1, v10, vcc
+v_mad_i32_i24 v105, v106, s77, v105
+v_readlane_b32 s79, v104, 2
+v_readlane_b32 s80, v105, 2
+v_readlane_b32 s81, v106, 2
+v_readlane_b32 s82, v105, 3
+v_readlane_b32 s83, v106, 3
+v_add_co_u32_e64 v104, vcc, v104, s78
+v_add_co_u32_e64 v105, vcc, v105, s77
+v_mov_b32_dpp v106, v106  quad_perm:[1,1,1,1] row_mask:0xf bank_mask:0xf
+v_mov_b32_dpp v104, v104  quad_perm:[1,1,1,1] row_mask:0xf bank_mask:0xf
+v_mov_b32_dpp v105, v105  quad_perm:[1,1,1,1] row_mask:0xf bank_mask:0xf
+s_mov_b32 s42, 0x80000000
+s_mov_b32 s43, 0x20000
+s_mov_b32 s46, 0x80000000
+s_mov_b32 s47, 0x20000
+s_mov_b32 s50, 0x80000000
+s_mov_b32 s51, 0x20000
+v_cmp_le_u32_e32 vcc, 0x100, v0
+s_cbranch_vccnz 7
+v_xor_b32_dpp v108, v0, v0  quad_perm:[1,3,2,2] row_mask:0xf bank_mask:0xf
+v_subrev_co_u32_e32 v108, vcc, 1, v108
+v_cvt_f16_i16_e32 v108, v108
+.long 0xd38f406c
```

Comment:
Such .long directives should be replaced with actual instruction or macro

Reply:
Thank you! Fixed in e5b26b2